src/PKSim.Core/Commands/SetExpressionProfileValueCommand.cs

```diff
@@ -27,7 +27,7 @@ protected override void UpdateParameter(IParameter parameter, IExecutionContext
       protected override void UpdateDependenciesOnParameter(IParameter parameter, IExecutionContext context)
       {
          base.UpdateDependenciesOnParameter(parameter, context);
-         if (!_updateIndividuals)
+         if (!_updateSimulationSubjects)
```

Comment:
of set to true (default), we have an automatic update of all depending simulation subjects.